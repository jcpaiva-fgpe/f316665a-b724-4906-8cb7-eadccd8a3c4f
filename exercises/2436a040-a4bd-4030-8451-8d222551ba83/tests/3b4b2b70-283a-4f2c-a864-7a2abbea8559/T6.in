1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0
0 0 0 1 0 1 0 0 0 0 0 1 0 1 0 0 0 0 1 0
0 1 0 0 1 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0
0 1 0 0 0 0 0 0 0 1 0 0 0 1 0 0 0 0 0 0
0 1 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0
0 1 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 1 1 1 1 0 0 0 0 0
0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0
1 0 0 0 0 0 0 1 1 0 1 0 0 0 0 1 0 1 0 0
0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
27
5 4 5 7 2 9
1 2 2 7 9 2
3 9 4 9 5 9
3 9 4 9 5 9
5 8 5 10 1 9
5 8 5 10 6 9
7 2 8 7 1 6
4 10 6 10 1 2
4 10 6 10 2 1
7 3 7 4 2 4
8 2 9 1 10 2
7 1 7 5 7 7
7 4 2 4 9 9
9 1 2 2 9 8
7 4 7 5 7 6
7 5 7 6 9 8
9 6 9 7 3 4
7 7 7 3 1 10
8 1 2 8 2 10
1 1 4 5 2 6
3 5 8 3 2 3
3 1 3 3 6 5
3 2 8 4 8 7
5 1 3 6 8 6
3 2 4 2 5 2
4 1 7 10 10 10
6 2 2 5 3 5
