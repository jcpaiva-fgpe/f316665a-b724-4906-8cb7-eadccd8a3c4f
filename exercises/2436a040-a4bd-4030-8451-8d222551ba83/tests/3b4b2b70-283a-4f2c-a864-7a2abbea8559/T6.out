Jogador 1: H1
Jogador 2: Agua
Jogador 1: Agua
Jogador 2: H1
Jogador 1: Agua
Jogador 2: Agua
Jogador 1: E1 H1
Jogador 2: Invalida
Jogador 2: H2
Jogador 1: E2 H1
Jogador 2: H3
Jogador 1: E1
Jogador 2: H1 C1
Jogador 1: H1 C1 S1
Jogador 2: Invalida
Jogador 2: C1
Jogador 1: H1
Jogador 2: Agua
Jogador 1: Agua
Jogador 2: H1 S1
Jogador 1: H1
Jogador 2: Agua
Jogador 1: Invalida
Jogador 1: Agua
Jogador 2: E3
Jogador 1: Agua
Jogador 2: E1 H1
Venceu o jogador 2
